app/dependencies.php

```diff
@@ -34,7 +34,7 @@
         return $renderer;
     };
     $container['db'] = function () {
-        $db = new \PDO('mysql:host=127.0.0.1; dbname=fishFood', 'root', 'password');
+        $db = new \PDO('mysql:host=DB; dbname=fishFood', 'root', 'password');
```

Comment:
Is this because you are trying to access it locally and this is the version that is hosted on the dev server? If so it might be worth just changing this locally rather than in the repo

Reply:
This fix and changing the scripts in the composer.json to  `"start": "php -S 0.0.0.0:8080 -t public",` were to get it working locally. This was the fix i had to do on every slim project from the academy which Mike showed me